CONTRIBUTORS.markdown

```diff
@@ -0,0 +1,50 @@
+Below you can find a list of people that contributed code to the IIS system (in order of appearance) before its move to GitHub (i.e., between September 2012 and September 2015) along with their:
```

Comment:
Since you are mentioning about IIS source code we could link to public svn here, although it will be mixed with all dnet modules.